### DISCUSSION TITLE
How do you know if you are using the optimal settings to create a transcript?

### DISCUSSION THREAD

**Discussion by ghost** [deepgram/community#434] [General help]
I am very sorry to ask such a simple question.  I am very new to coding and STT applications.  This is a section of script that I have created.  I am running it in Google Colab.  I think that I have the settings correct, however I just am not sure.  It is very important that the transcript I am creating is as accurate as possible.  Any help with this would be greatly appreciated.

# Section Four
from deepgram import Deepgram
import asyncio, json, os

# Deepgram API key 
dg_key = 'my key'

# Initialize Deepgram SDK
dg = Deepgram(dg_key)

# Audio settings
MIMETYPE = 'flac'  
DIRECTORY = '/content/'

# Transcription options
options = {
  "punctuate": True,
  "diarize": True, 
  "speaker_labels": True,
  "ner": True,
  "language": 'en-US',
  "model": 'nova-2-ea', 
  "filler_words": True,
  "sample_rate": 16000,
  "utterances": True, 
  "smart_words": True,
  "numerals": True,
  "tier": 'nova'
}

# Transcribe audio files
def transcribe():
  
  audio_files = os.listdir(DIRECTORY)

  for audio_file in audio_files:

    if audio_file.endswith(MIMETYPE):

      with open(f"{DIRECTORY}/{audio_file}", "rb") as f:
      
        source = {"buffer": f, "mimetype":'audio/'+MIMETYPE}
      
        try:
          res = dg.transcription.sync_prerecorded(source, options)
        
          with open(f"./{audio_file[:-4]}.json", "w") as transcript:
          
            json.dump(res, transcript, indent=4)
        
        except Exception as e:
          print(f"Error transcribing {audio_file}: {e}")

if __name__ == '__main__':
  transcribe()

**Comment by DamienDeepgram** [comment]
In order to help you can you share what the particular issue you are seeing when running this code?

From a quick glance I would recommend the following settings

> options = {
"diarize": True,
"ner": True,
"language": 'en-US',
"model": 'nova-2-ea',
"filler_words": True,
"utterances": True,
"smart_words": True,
}
